Dockerfile

```diff
@@ -1,9 +1,14 @@
-FROM python:2.7-alpine
+FROM ubuntu:xenial
```

Comment:
Curious, why don't we use the `python:2.7` image (non Alpine)? Then we wouldn't need to install Python/pip.

Reply:
Can give that a whirl; thanks for the suggestion/question!

Reply:
Fixed